bitnami/dataplatform-bp2/templates/_helpers.tpl

```diff
@@ -0,0 +1,8 @@
+{{/*
+Return the proper service name for a subcomponent
+Usage:
+{{ include "subcomponent.service.name" ( dict "componentName" "name" "context" $ ) }}
+*/}}
+{{- define "subcomponent.service.name" -}}
+{{- printf "%s-%s" .context.Release.Name .componentName -}}
```

Comment:
Shouldn't you truncate these string to limit to 63 chars because of the Kubernetes name fields limitation?